README.md

```diff
@@ -83,7 +83,6 @@ youtube2zim --api-key "<your-api-key>" --type user --id "Vsauce" --name "vsauce"
 
 ## Notes
 
-* Your API_KEY is subject to usage quotas (10,000 requests/day) so use `--only_test_branding` when adjusting parameters and branding to not *waste your quota*.
```

Comment:
Even if parameter is gone, I would keep the warning with something like "Your API_KEY is subject to usage quotas (10,000 requests/day by default) so be careful to not *waste your quota*, especially when scraping or testing big channels.

Reply:
Updated in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4).